ts5/ts5/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+
+namespace ts5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст на английском языке");
+
+            string EngSt = Console.ReadLine();
+
+            string LeerSt = FrEngToLeer(EngSt);
```

Comment:
Локальные переменные в C# принято назвать с маленькой буквы. А этот способ написания текста называется Lee**t**
```suggestion
            string engSt = Console.ReadLine();

            string leetSt = FrEngToLeet(EngSt);
```